engine/consensus/compliance/core.go

```diff
@@ -287,7 +295,7 @@ func (c *Core) processBlockAndDescendants(proposal *flow.Block) error {
 		return nil
 	}
 	for _, child := range children {
-		cpr := c.processBlockAndDescendants(child.Message)
+		cpr := c.processBlockAndDescendants(originID, child.Message)
```

Comment:
## ⚠️ potentially faulty attribution (?)

same reasoning as in [my comment on `compliance.Core`]()